src/titiler/core/titiler/core/factory.py

```diff
@@ -367,6 +370,7 @@ def info_geojson(
             with rasterio.Env(**env):
                 with self.reader(src_path, **reader_params) as src_dst:
                     return Feature(
+                        type="Feature",
```

Comment:
This change is unrelated to the PR goal but it will prevent breaking change with the next version of geojson-pydantic